server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -302,16 +302,23 @@
               }
             }
 
+            const _header = () => {
```

Comment:
Can this be given a more meaningful name?

Reply:
of course

Reply:
done :D